pyScienceMode2/rehastim_interface.py

```diff
@@ -5,30 +5,26 @@
 from typing import Tuple
 import time
 from pyScienceMode2.acks import *
-from pyScienceMode2 import Channel
+from pyScienceMode2 import channel
 from pyScienceMode2.utils import *
 from pyScienceMode2.sciencemode import RehastimGeneric
 from pyScienceMode2.motomed_interface import _Motomed
+from sciencemode_p24 import sciencemode
 
 
-class Stimulator(RehastimGeneric):
+class Stimulator2(RehastimGeneric):
```

Comment:
As I said I think name should be changed

Reply:
Done.